test/test_core.py

```diff
@@ -6187,6 +6187,7 @@ def test_signals(self):
     'sigint': (signal.SIGINT, 128 + signal.SIGINT, True),
     'sigabrt': (signal.SIGABRT, 7, False)
```

Comment:
Ah.. I imagine what is happening here is that `signal.SIGABRT` has a different value on windows .. looks like its 6 on linux and maybe 22 on windows?

Reply:
Heh. Might be another reason to export signal numbers via cDef if they're not the same on different platforms.

Reply:
cDef stuff will be the same since they expose the values from running the code in wasm, not the host.